app/templates/create.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{%block content%}
+<div class="container-fluid  min-vh-100 bg-light">
+  <h1 class="my-3 text-center">Create a Challenge</h1>
+  
+  <form class="container my-1 challenge-border bg-white" id="createForm" method="put">
```

Comment:
method should be POST, not put (i think case matters)

Reply:
I've changed it to post now, but still in lowercase, as this is the convention in HTML5.